mentat/conversation.py

```diff
@@ -254,14 +266,42 @@ async def get_model_response(self) -> list[FileEdit]:
                     p.get("text", "") for p in prompt if p.get("type") == "text"
                 ]
                 prompt = " ".join(text_prompts)
+            max_tokens = self._get_max_tokens()
+            if max_tokens is None:
```

Comment:
Could we move this if statement outside the try catch and to the top of the function?